jitsu/helm/jitsu/templates/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
             - name: config
               containerPort: 8000
               protocol: TCP
+            - name: test
```

Comment:
is `test` the right name for the port or does it serve a different role within jitsu?

Reply:
I think a better name will be readiness/liveness. I used "test" as this is used by the probes to initially test the url. As this was failing (port was not exposed), k8 was killing the deployment.

Reply:
how about `health`?

Reply:
I changed to health. I was also supposed to change it in here:
          livenessProbe:
            httpGet:
              path: /
              port: health

but on the deployment I'm getting this error. 
Liveness probe errored: strconv.Atoi: parsing "health": invalid syntax

Reverted to http.